src/commands/dev-env-import-sql.js

```diff
@@ -79,7 +79,7 @@ export class DevEnvImportSQLCommand {
 		}
 
 		const fd = await fs.promises.open( resolvedPath, 'r' );
-		const importArg = [ 'db', '--disable-auto-rehash' ].concat(
+		const importArg = [ 'db', '--disable-auto-rehash', '--skip-plugins', '--skip-themes' ].concat(
```

Comment:
```suggestion
		const importArg = [ 'db', '--disable-auto-rehash' ].concat(
```

This command invokes `mysql`, not `wp`.

Reply:
my bad